src/entities/raw-data-providers/cardano-explorer-api/db-queries.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+import squel from 'squel'
+
+export const sql = squel.useFlavour('postgres')
+
+export const GET_BEST_BLOCK_NUM = sql.select()
+  .from('"Block"')
+  .field('id', 'hash')
+  .field('number', 'height')
+  .field('"epochNo"', 'epoch')
```

Comment:
PostgreSQL converts all table column names into lowercase, unless quoted. https://stackoverflow.com/questions/20878932/are-postgresql-column-names-case-sensitive

Reply:
maybe add this as a code comment somewhere?